### PR TITLE
ENH: Try cross compiling

### PR DESCRIPTION
**Pull request by bashtage**
Checklist
* [X] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [X] Bumped the build number (if the version is unchanged)
* [X] Reset the build number to `0` (if the version changed)
* [X] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [X] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by bashtage** [comment]
@conda-forge-admin, please rerender

**Comment by bashtage** [comment]
@conda-forge-admin, please rerender

**Comment by bashtage** [comment]
@conda-forge-admin, please rerender

**Comment by khwilson** [comment]
Oh hi! I added this feedstock for a specific project that I was working on. If you'd like to take over this feedstock, I'm very happy to pass it on. :-)

**Comment by bashtage** [comment]
I'm happy to provide maintenance.  Can you merge this?  Thie drone is passing (did the first 2 runs, you can see if you click details) it is just very slow.

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1 +1 @@
-* @khwilson
\ No newline at end of file
+* @bashtage @khwilson
```
Comment (khwilson) [inline .github/CODEOWNERS:1]: I think this needs to be in list form:

```
* @bashtage
* @khwilson
```

File: README.md [hunk 2]
```diff
@@ -9,15 +9,27 @@ Feedstock license: [BSD-3-Clause](https://github.com/conda-forge/linearmodels-fe
 
 Summary: Instrumental Variable and Linear Panel models for Python
 
-Development: http://github.com/bashtage/linearmodels
+Development: https://github.com/bashtage/linearmodel
```
Comment (khwilson) [inline README.md]: Typo. `/linearmodels`

File: recipe/meta.yaml [hunk 3]
```diff
@@ -76,9 +79,14 @@ about:
...
   license_file: LICENSE.md
   summary: Instrumental Variable and Linear Panel models for Python
-  doc_url: https://bashtage.github.io/linearmodels/
-  dev_url: http://github.com/bashtage/linearmodels
+  description: |
+    Linear (regression) models for Python. Extends statsmodels with Panel
+    regression, instrumental variable estimators, system estimators and
+    models for estimating asset prices.
+  doc_url: https://bashtage.github.io/linearmodels
+  dev_url: https://github.com/bashtage/linearmodel
```
Comment (khwilson) [inline recipe/meta.yaml]: Typo

File: recipe/meta.yaml [hunk 4]
```diff
@@ -10,34 +10,37 @@ source:
...
     - python
-    - scipy >=1
-    - statsmodels >=0.9
+    - scipy >=1.2
+    - statsmodels >=0.11
   run:
     - mypy_extensions >=0.4
-    - numpy >=1.15
-    - pandas >=0.23
+    - {{ pin_compatible('numpy') }}
```
Comment (khwilson) [inline recipe/meta.yaml:40]: Maybe this is the thing restricting?

**Review by khwilson — COMMENTED** [review]
Happy to merge, though there's a few small typos. Notably, this commit only seems to be testing on numpy 1.18 and 1.19 but numpy's at 1.21 now. Perhaps this is due to a `pin_compatible` in the `meta.yml` ?

Just want to make sure that that's correct

**Comment by khwilson** [comment]
Just a couple typos (if this is super urgent, I'm happy to merge and do another PR for the typos) :-)

And one quick question re the versions of numpy that are actually being tested. Just want to make sure that that's correct!

File: .github/CODEOWNERS (hunk 1 above)
Comment (bashtage) [reply .github/CODEOWNERS:1]: I think the current is correct:


See, e.g., 

https://github.com/conda-forge/statsmodels-feedstock/blob/master/recipe/meta.yaml

**Comment by bashtage** [comment]
@conda-forge-admin, please rerender

**Comment by bashtage** [comment]
Can wait a bit to merge.  I'm trying to follow other recipes I see around like pandas and statsmodels.

**Comment by bashtage** [comment]
@conda-forge-admin, please rerender

**Comment by bashtage** [comment]
Looks like pyhdfe needs a feedstock first 👎

**Comment by khwilson** [comment]
Boooo

**Comment by bashtage** [comment]
pyhdfe package at conda-forge/staged-recipes/pull/16443

**Comment by bashtage** [comment]
@conda-forge-admin, please rerender

**Comment by khwilson** [comment]
I think it takes a little time for everything to percolate because it seems to indicate that it's still waiting for pyhdfe.

By the way, for high dimensional fixed effects, you may be interested in using the tricks from Laurent Berge's R package. Giorgos Zervas put together a simple torch implementation here https://github.com/gzervas/torchfe

Vendoring this tiny model and relying on torch maintainers might be easier than getting pyhdfe to work :-)

**Comment by bashtage** [comment]
@conda-forge-admin, please rerender

**Comment by bashtage** [comment]
@khwilson I think it is ready for merge. Thanks for the suggestion.

File: .github/CODEOWNERS (hunk 1 above)
Comment (khwilson) [reply .github/CODEOWNERS:1]: fascinating

**Comment by khwilson** [comment]
LGTM! :shipit:

**Pull request merged**